src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command_json.py

```diff
@@ -702,7 +702,9 @@ def on_source_request(self, py_db, request):
         response_args = {'body': body}
 
         if content is None:
-            if server_filename:
+            if source_reference == 0:
```

Comment:
Can you double-check if there are tests for this case? (even if only in the `ptvsd` side)

Reply:
Updated `test_with_path_mappings` to test this case.